src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -250,23 +272,8 @@ public PrivilegesEvaluatorResponse evaluate(
             action0 = PutMappingAction.NAME;
         }
 
-        final PrivilegesEvaluatorResponse presponse = new PrivilegesEvaluatorResponse();
+        PrivilegesEvaluatorResponse presponse = new PrivilegesEvaluatorResponse();
```

Comment:
will this be mutated at some point?

Reply:
Not at the moment, but it will be in #4380. But for now, I can revert that change.